### PR TITLE
[IMP] inventory: print shipping label

### PR DESCRIPTION
**Pull request by Felicious**
## Summary
1. Overhaul 8 year old doc about generating shipping labels
2. Merge after [3rd party shipping](https://github.com/odoo/documentation/pull/8634) doc because I reused the images :eyes: 

FWport: yes!

[Task](https://www.odoo.com/web#id=3576553&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by brse-odoo** [comment]
@Felicious I'll review this right away!

**Review by brse-odoo — APPROVED** [review]
@Felicious All done reviewing this, great job! Approving with very few suggestions, let me know if you have any questions, Thanks!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels/sample-label.png
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels/sample-label.png:1]: OH MY GOODNESS. I'll blur out the address! Thanks for catching that!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:98]: For! Thanks for pointing out the distinction

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:99]:
```suggestion
drop-down menu. Clicking :guilabel:`Get Rate` displays the shipping :guilabel:`Cost` for the customer
via the third-party carrier in the :guilabel:`Cost` field.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:17]: @odoo/us-doc-review, I wanted to get your opinion on how short to keep this Configuration section.

Technically, the configuration of this doc is the exact same as the third_party_shipper's (the PR that's supposed to be merged before this one). The only difference is that we set the "integration level" to "Get Rate and Create Shipment" in the delivery method section. 

I chose to keep the shipping connector installation part in this doc because it made sense, but it can be taken out. Should I keep the "Install shipping connector" section in the doc, or should I just link to the other doc?

I decided to not include the source address and product weights configuration in the doc, as this doc and the third_party_shipper one starts to become really similar. Instead, in the "Add shipping on quotation" section, I mention that the source address and product weights must be configured for the shipping connector to work properly. Do you think mentioning it there is enough? Would love to know your thoughts!

Comment (samueljlieber) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:17]: Hi @Felicious, this is a really good conversation to have– I like that you are trying to reduce writing duplicate information by leveraging the modular nature of docs. After looking at both this PR and #8634, I think that you can remove a lot of the same/similar content you have in this PR (#8658), here is my reasoning:

In the **Configuration** section of #8634, you already detail (with an important admonition) the **Integration Level** needed for generating shipping labels. I think this is a perfect use of an admonition and internal reference.

So for the **Configuration** section of this PR (#8658), I think you can get by with just stating something like the following, removing the **Install shipping connector**, **Delivery method**, and **Production environment** sections from this PR:
```
Configuration
=============

To generate labels for a third-party shipping carrier, first :doc:`install the third party shipping connector
<../setup_configuration/third_party_shipper>`. Then configure and activate the :ref:`delivery method
<inventory-shipping-receiving-configure-delivery-method>`, being sure to set the :guilabel:`Integration
Level` to :guilabel:`Get Rate and Create Shipment` to generate shipping labels. Finally, provide the
company's :ref:`source address <inventory/shipping_receiving/configure-source-address>` and
:ref:`product weights <inventory/shipping_receiving/configure-weight>`. 

.. seealso::
   :doc:`../setup_configuration/third_party_shipper`
```

Lastly, I think your important admonition in the **Add shipping on quotation** works well, and mentioning it there is enough IMO.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:28]:
```suggestion
Under the :guilabel:`Shipping Connectors` section, tick the third-party shipping carrier's
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:41]:
```suggestion
On the :guilabel:`Shipping Methods` page, select the desired delivery method from the list. For details on
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:42]:
```suggestion
configuring the third-party shipping method, go to:
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:51]:
```suggestion
Level` field to ensure shipping labels are printed.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:61]:
```suggestion
set it to :guilabel:`Production Environment`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:64]:
```suggestion
   Setting the delivery method to :guilabel:`Production Environment` creates **real** shipping labels, and users
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:65]:
```suggestion
   are at risk for being charged through their carrier account (e.g. UPS, FedEx, etc.) **before** users
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:67]:
```suggestion
   delivery method to :guilabel:`Production Environment`.
```
I am suggesting the addition of 'Environment' -- and adding capitalization -- because that's how it appears on the UI/screen

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:76]:
```suggestion
Tracking labels are generated after a delivery order (DO) is validated. Begin by creating a
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:77]: I genuinely appreciate the concise nature of this instruction, however, do you think a quick mention of where they can do this would be helpful? Like, "Open the :menuselection:`Sales app`, and proceed to..."

Or something like that? What do you think?

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:83]:
```suggestion
app --> Orders --> Quotations`, clicking :guilabel:`New`, and filling out the quotation form. Then, click the :guilabel:`Add Shipping` button in the bottom-right
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:91]:
```suggestion
drop-down menu. Clicking :guilabel:`Get Rate` displays the shipping cost for the customer, via the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:119]:
```suggestion
On a delivery order form, after the items in the order have been packed, click :guilabel:`Validate` to
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:120]:
```suggestion
get the shipping carrier's tracking number, and generate the shipping label.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:77]: I'm not sure if specifying how to make a quotation is necessary in this opening paragraph, as the "Add shipping on quotation" section goes into detail on how to do that (:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:15]:
```suggestion
connector <../setup_configuration/third_party_shipper>`. Then, configure and activate the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:98]:
```suggestion
   Create or select an existing delivery order by going to the :menuselection:`Inventory` app, and
```

**Review by ksc-odoo — APPROVED** [review]
@Felicious -- looks good to me! Just a couple SUPER SMALL suggestions that require your attention 👍

**Comment by Felicious** [comment]
Hi @samueljlieber ! This PR has passed final review and is ready for your tech review, whenever you get the chance (:

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious! Great job improving the content on this Inventory doc. Your changes in this PR look good to me– can you just double check the images I marked as they seem to be extra and unused, if so please remove before merging! Approving and delegating to you.
.....
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**
